app/src/main/res/layout/item_data_transaction2.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Esse layout não é igual ao `app/src/main/res/layout/item_data_transaction.xml`? se for, mantem só 1